cmd/root.go

```diff
@@ -45,6 +46,8 @@ func NewRootCmd(out io.Writer, args []string) *cobra.Command {
 	flags.Parse(args)
 	settings = new(EnvSettings)
 
+	settings.KubeContext = os.Getenv("HELM_KUBECONTEXT")
```

Comment:
Can you add a comment here explaining about how plugin framework handles  `kubeconfig` and `kube-context` flags

Reply:
Done.